src/core/a-scene.js

```diff
@@ -93,24 +89,12 @@ var AScene = module.exports = registerElement('a-scene', {
         var resizeCanvas = this.resizeCanvas.bind(this);
         this.setupKeyboardShortcuts();
         this.attachFullscreenListeners();
```

Comment:
I pushed my updated changes to this PR. Moved fullscreen stuff to https://github.com/aframevr/aframe/pull/776/files#diff-344cbe83c2f901ba50d1f8824d89ccfeR18

So you want the use case to always force stereo and completely ignore fullscreen events?